docs/core/setup/index.md

```diff
@@ -0,0 +1,453 @@
+---
+title: Install .NET Core on Windows, Linux, and macOS
+description: Learn where and what to install for .NET Core on Windows, Linux, and macOS versions. Discover the dependencies required to develop, deploy, and run .NET Core apps.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Install .NET Core
+
+This article provides information on downloading .NET Core and what is required to install .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and apps.
+
+.NET Core is made up of two components, the runtime and the SDK. The **runtime** is required to run a .NET Core app, when that app doesn't include the runtime. The **SDK** is required to build .NET Core apps and libraries.
+
+There are different ways to develop .NET Core apps:
+
+::: zone pivot="os-windows"
+
+- [Visual Studio](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2019)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+::: zone pivot="os-linux"
+
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+::: zone pivot="os-macos"
+
+- [Visual Studio for Mac](https://visualstudio.microsoft.com/vs/mac/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link)
+- [Visual Studio Code](https://code.visualstudio.com/)
+- [Command-line with your favorite editor](../tutorials/using-with-xplat-cli.md)
+
+::: zone-end
+
+You can download a standalone installer for each OS from the following links:
+
+- [.NET Core 3.0 downloads](https://dotnet.microsoft.com/download/dotnet-core/3.0).
+- [.NET Core 2.2 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.2).
+- [.NET Core 2.1 downloads](https://dotnet.microsoft.com/download/dotnet-core/2.1).
+
+It's important to understand the [.NET Core support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) when choosing which version of .NET Core to develop apps for.
+
+## Check if .NET Core installed
```

Comment:
```suggestion
## Check if the .NET Core SDK installed
```

Reply:
It's a bit odd to start an install article with check if the SDK is already installed